evaluation/tasks/auto_task.py

```diff
@@ -7,18 +7,21 @@
 
 from evaluation.models.loader import load_model
 from evaluation.utils.io import load_json, save_json
+from argparse import ArgumentParser
 
 
 class AutoTask(ABC):
     def __init__(
-        self,
-        model: PreTrainedModel,
-        tokenizer: PreTrainedTokenizerFast,
-        device: torch.device,
-        english_only: bool,
-        data_dir: Optional[str] = None,
+            self,
+            args: ArgumentParser,
+            model: PreTrainedModel,
+            tokenizer: PreTrainedTokenizerFast,
+            device: torch.device,
+            english_only: bool,
+            data_dir: Optional[str] = None,
```

Comment:
nit: Follow standard convention and move `args` to the end of the list of keyword arguments
```suggestion
            self,
            model: PreTrainedModel,
            tokenizer: PreTrainedTokenizerFast,
            device: torch.device,
            english_only: bool,
            args: ArgumentParser,
            data_dir: Optional[str] = None,
```